types/kagekiri.d.ts

```diff
@@ -12,3 +12,4 @@ export function getElementsByTagName(tagName: string, context?: Node): Element[]
 export function getElementsByTagNameNS(namespaceURI: string, localName: string, context?: Node): Element[];
 export function getElementById(id: string, context?: DocumentOrShadowRoot): Element | null;
 export function getElementsByName(name: string, context?: DocumentOrShadowRoot): Element[];
+export function matches(selector: string, context?: Node): boolean;
```

Comment:
`context` is not optional here, correct? I think we should remove the `?`.

Reply:
You're right! It kept bugging me that something was off.